package.json

```diff
@@ -42,10 +42,12 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-react-native": "^3.10.0"
+    "eslint-plugin-react-native": "^3.10.0",
+    "@typescript-eslint/eslint-plugin": "^4.15.1",
+    "@typescript-eslint/parser": "^4.15.1",
+    "eslint": "^7.20.0"
```

Comment:
we should remove eslint as dependency otherwise it will be installed with our config. We only use it in development to test our rules. Please add it as devDependency

Reply:
Add this one to the package.json, too please! 
```json
  "peerDependencies": {
        "eslint": ">=7.12.0"
    },
```